packages/expo-video/src/VideoView.web.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useRef, forwardRef, useImperativeHandle } from 'react';
+
+import { VideoPlayer } from './VideoPlayer';
+import { VideoViewProps } from './VideoView.types';
+class VideoPlayerImpl implements VideoPlayer {
```

Comment:
```suggestion

class VideoPlayerImpl implements VideoPlayer {
```

Reply:
Suggestion to consider: we could use `VideoPlayerWeb` name to indicate that this is an implementation for web